packages/docusaurus-init/templates/classic/docs/markdown-features.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Markdown Features
+---
+
+In this page, we will see some of the tools that will help you to build your documentation site with Docusaurus.
+
+## Markdown headers
+
+You will see markdown headers a lot when you're building your documentations. They define your documentation content in fields. For example:
+
+```md
+---
+id: slash
+title: Slash, Our Mascot!
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Say Hello to Slash!
+description: We named our mascot Slash because most documentations start with //
+keywords:
+  - docs
+  - docusaurus
+  - mascot
+---
+```
+
+## Referencing other documents
+
+You can reference to another document file that you have in your documentation by using the file name of that documentation.
+
+For example, if you want to reference `create-a-page.md`:
+
+```md
+I am referencing [this](create-a-page.md) tutorial.
```

Comment:
displaying a link using an absolute path like `[this](/create-a-page)` can also be useful